bundles/org.openhab.binding.cp750/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="cp750"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="connection">
+		<label>CP750 Binding Thing</label>
+		<description>Sample thing for CP750 Binding</description>
+
+		<channels>
+			<channel id="channel-fader" typeId="fader"/>
+			<channel id="channel-mute" typeId="mute"/>
+			<channel id="channel-input" typeId="input"/>
+			<channel id="channel-analog" typeId="input-analog"/>
+			<channel id="channel-dig1" typeId="input-dig1"/>
+			<channel id="channel-dig2" typeId="input-dig2"/>
+			<channel id="channel-dig3" typeId="input-dig3"/>
+			<channel id="channel-dig4" typeId="input-dig4"/>
+			<channel id="channel-nonsync" typeId="input-non_sync"/>
+			<channel id="channel-mic" typeId="input-mic"/>
+			<channel id="channel-version" typeId="version"/>
+		</channels>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the device</description>
+			</parameter>
+			<parameter name="port" type="integer" min="1" max="65535">
+				<label>Port</label>
+				<default>61408</default>
+				<description>Port to access the device</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" unit="s" min="0">
+				<label>Refresh Interval</label>
+				<description>Interval the device is polled in sec.</description>
+				<default>5</default>
+			</parameter>
+			<parameter name="reconnectInterval" type="integer" unit="s" min="0">
+				<advanced>true</advanced>
+				<label>Retry interval</label>
+				<description>Reconnect interval in seconds after connection failure (0 = disabled)</description>
+				<default>10</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Sample Channel Type -->
+	<channel-type id="fader">
+		<item-type>Dimmer</item-type>
+		<label>Fader channel</label>
```

Comment:
Not 100% sure, but i don't think the channel label should contain the word 'Channel', same as switch (see other comment)